src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
```

Comment:
you already added box-sizing in `*` selector, delete here and 
https://github.com/mate-academy/layout_catalog/pull/3606/files#diff-efc436905cb807e5305d392062b3125e8ef099b2665dfbb262301b5e1355a96eR72